src/app.ts

```diff
@@ -157,34 +156,21 @@ export type ContextData = {
 
 const fedifyKv = await KnexKvStore.create(client, 'key_value');
 
-const eventBus = new EventEmitter();
-
-let queue: MessageQueue | undefined;
+let queue: GCloudPubSubPushMessageQueue | undefined;
 
 if (process.env.USE_MQ === 'true') {
     logging.info('Message queue is enabled');
 
     try {
-        const host = String(process.env.MQ_PUBSUB_HOST) || undefined;
-        const emulatorMode = process.env.NODE_ENV !== 'production';
-        const projectId = String(process.env.MQ_PUBSUB_PROJECT_ID) || undefined;
-        const topicName = String(process.env.MQ_PUBSUB_TOPIC_NAME);
-        const subscriptionName = String(
-            process.env.MQ_PUBSUB_SUBSCRIPTION_NAME,
-        );
+        queue = await createMessageQueue(logging, {
+            pubSubHost: String(process.env.MQ_PUBSUB_HOST) || undefined,
```

Comment:
Why are we using `String` here? `process.env` should be a `Dict<string>` which means the properties will always be a `string` or `undefined`

Reply:
Yep no need for this, thanks 👍

Fixed in https://github.com/TryGhost/ActivityPub/pull/180/commits/db60dc3e5a7d0e345c52068a6d8d1f31a1f998f2